src/tests/ftest/launch.py

```diff
@@ -711,6 +712,13 @@ def replace_yaml_file(yaml_file, args, tmp_dir):
     # Return the untouched or modified yaml file
     return yaml_file
 
+def generate_certs():
+    """Generate the certificates for the test."""
+    daos_test_log_dir = os.environ["DAOS_TEST_LOG_DIR"]
+    certs_dir = os.path.join(daos_test_log_dir, "certs")
+    subprocess.call(["/usr/bin/rm", "-rf", certs_dir])
+    subprocess.call(["../../../../lib64/daos/certgen/gen_certificates.sh"])
```

Comment:
This will fail in RPM installations due to the permissions of the ftest directory:
```
[pahender@wolf-10 ~]$ ll /usr/lib/daos/TESTING/
total 12
drwxr-xr-x 18 root root 4096 Jul  1 21:15 ftest
```
The ideal solution is to have `gen_certificates.sh` accept `daos_test_log_dir` as an input in which to create the files.